src/Query.js

```diff
@@ -263,13 +263,21 @@ export default Component.extend({
       cB = cA
       cA = temp
     }
-    if (cA < cB) {
-      return -1
-    } else if (cA > cB) {
-      return 1
+    /* Fix: compare by using collator */
+    let isNumeric = false
+    if (utils.isNumber(cA) || utils.isNumber(cB)) {
+      isNumeric = true
+    }
+    const collator = new Intl.Collator(locale, {
```

Comment:
Will this throw if `Intl` is not defined in an environment?

Reply:
Ok i will check for undefined and add it the default behavior, that would help?

Reply:
Yeah, definitely need that. This won't run in Node.js env at all

Reply:
@crobinson42 looks like `Intl` API is available in `Node.js` after all - I've checked it starting from version `8.11.1` (oldest one on my machine) and it works just fine. I am using `Node` installed by `nvm` on `macOS`.

Reply:
I suppose it wouldn't hurt to only support Node >= 8 in v4 release.. I notice a lot of popular npm packages are dropping support for old Node.js versions. What do you guys think?

Reply:
We will still need either a polyfill or a graceful failover for `Intl` api: https://caniuse.com/#search=Intl

Reply:
I think it's a good idea to drop Node below 8.

As for `Intl`, actually `Collator.compare` part of this API is pretty much widely supported: https://caniuse.com/#feat=mdn-javascript_builtins_intl_collator_compare

I wouldn't say we need polyfill, unless we want to support IE 10 and below (which I would drop as well)

Reply:
I agree that there is no need to support IE 10 if newer browser works, i already added support for edge by force setting locale...
Other than that i can add to the docs and help write a test for the #540 and call it a new day?

Reply:
I agree that there is no need to support IE 10 if newer browser works, i already added support for edge by force setting locale...
Other than that i can add to the docs and help write a test for the #540 and call it a new day?

Reply:
@wick Sounds good 👍

Reply:
updated the doc, not sure where to put the available since version :)